### ISSUE TITLE
RegDB的Rank-1和mAP比论文报告的少了近20个百分点

### ISSUE THREAD

**Issue by hnlxy** [nengdong96/CSDN#14]
当前的RegDB在RTX 4090上的评估结果如下，Rank-1和mAP比论文报告的少了近20个百分点，如果RegDB有效果更好的超参数，作者是否可以提供一下，非常感谢！
Namespace(auto_resume_training_from_lastest_step=True, batch_size=32, cuda='cuda', dataset='regdb', eval_epoch=1, gall_mode='single', img_h=288, img_w=144, lambda1=0.15, lambda2=0.05, lambda3=0.1, learning_rate=0.0003, loss=1, max_save_model_num=1, milestones=[40, 70], mode='test', num_pos=4, num_workers=8, output_path='regdb/trial1', pid_num=395, regdb_data_path='../../datasets/RegDB/', regdb_test_mode='v-t', resume_test_model=101, resume_train_epoch=-1, seed=1, stage1_batch_size=32, stage1_learning_rate=0.0003, stage1_lr_min=1e-06, stage1_train_epochs=60, stage1_warmup_epochs=5, stage1_warmup_lr_init=1e-05, stage1_weight_decay=0.0001, stage2_learning_rate=0.0003, sysu_data_path='../../datasets/SYSU-MM01/', test_mode='all', total_train_epoch=120, trial=1, weight_decay=0.0005)
Time: 2026-07-15 18:00:19; Test on Dataset: regdb, 
mINP: 0.5589078614545755 
mAP: 0.7067291504965686 
 Rank: [0.76796114 0.8106796  0.83980584 0.8592233  0.87281555 0.8864078
 0.9        0.9121359  0.9199029  0.9291262  0.9349515  0.93883497
 0.9470874  0.95       0.9519417  0.95485437 0.957767   0.9611651
 0.96359223 0.965534  ]
